v4/weaviate/graphql/rawgql_test.go

```diff
@@ -0,0 +1,30 @@
+package graphql
+
+import "testing"
+
+func TestRawGQLBuilder_build(t *testing.T) {
+	type fields struct {
+		query string
+	}
+	tests := []struct {
+		name   string
+		fields fields
+		want   string
+	}{
+		{
+			name: "with query",
+			fields: fields{
+				query: "query { Get { Things { Thing { name } } } }",
+			},
+			want: "query { Get { Things { Thing { name } } } }",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			e := &RawGraphQLQueryBuilder{query: tt.fields.query}
+			if got := e.build(); got != tt.want {
+				t.Errorf("RawGQLBuilder.build() = %v, want %v", got, tt.want)
+			}
```

Comment:
Would be good to have some error cases added

Reply:
Integration tests added